timeseries/setup.py

```diff
@@ -24,16 +24,16 @@
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
     "joblib~=1.1",
-    "numpy",
-    "scipy",
-    "pandas",
+    "numpy",  # version range defined in `core/_setup_utils.py`
+    "scipy",  # version range defined in `core/_setup_utils.py`
+    "pandas",  # version range defined in `core/_setup_utils.py`
     "statsmodels~=0.13.0",
     "gluonts~=0.12.0",
```

Comment:
`~=0.1.3.0` -> `~=0.13` ?

Reply:
This PR is not touching the version ranges to minimize potential merge conflicts with other PRs. A follow-up PR will be sent to clean up the ranges.